### ISSUE TITLE
Should we use table titles and not numbers in markdown files?

### ISSUE THREAD

**Issue by davidlmobley** [MobleyLab/benchmarksets#54]
@nhenriksen @GHeinzelmann - I notice both of you reference (in your markdown files, which are great!) specific table numbers in the paper. I wonder if we should be referencing tables by title rather than by number. Otherwise, if we change things in the paper such that tables auto-renumber, then the table references will all be wrong and someone will have to fix them. If we just refer to them by title then we won't have to remember to change.

Thoughts?

**Comment by GHeinzelmann** [comment]
I agree, we could say something like the BRD4(1) associated table in the main paper, to avoid numbering them.

**Comment by nhenriksen** [comment]
Agreed.  will edit.

**Comment by davidlmobley** [comment]
@GHeinzelmann - works for me, or you can give the full title of it in quotes.

**Comment by GHeinzelmann** [comment]
OK I think that is better.

**Comment by nhenriksen** [comment]
Hmm ... I actually don't see anywhere that I refer to a specific table number.  My README's say "CD Set 1" and "CD Set 2".

**Comment by nhenriksen** [comment]
Oh, now I understand.  In the master branch, the README has table numbers.  That should be taken care of in the nieldev branch.
